src/collective/js/jqueryui/jquery-ui-1.13.2.custom/jquery-ui.min.js

```diff
@@ -0,0 +1,6 @@
+/*! jQuery UI - v1.13.2 - 2023-02-09
+* http://jqueryui.com
+* Includes: widget.js, position.js, data.js, disable-selection.js, focusable.js, form-reset-mixin.js, jquery-patch.js, keycode.js, labels.js, scroll-parent.js, tabbable.js, unique-id.js, widgets/draggable.js, widgets/droppable.js, widgets/resizable.js, widgets/selectable.js, widgets/sortable.js, widgets/accordion.js, widgets/autocomplete.js, widgets/button.js, widgets/checkboxradio.js, widgets/controlgroup.js, widgets/datepicker.js, widgets/dialog.js, widgets/menu.js, widgets/mouse.js, widgets/progressbar.js, widgets/selectmenu.js, widgets/slider.js, widgets/spinner.js, widgets/tabs.js, widgets/tooltip.js, effect.js, effects/effect-blind.js, effects/effect-bounce.js, effects/effect-clip.js, effects/effect-drop.js, effects/effect-explode.js, effects/effect-fade.js, effects/effect-fold.js, effects/effect-highlight.js, effects/effect-puff.js, effects/effect-pulsate.js, effects/effect-scale.js, effects/effect-shake.js, effects/effect-size.js, effects/effect-slide.js, effects/effect-transfer.js
```

Comment:
Yes. All is there. How it is by default on https://jqueryui.com/download/
Since the registry based configuration is removed I went with the simplest solution.
Updated selection as suggested - see commit https://github.com/collective/collective.js.jqueryui/commit/1ff1cd8fa1269a133c89cae30a733eb84f086f6a and https://github.com/collective/collective.js.jqueryui/blob/1ff1cd8fa1269a133c89cae30a733eb84f086f6a/src/collective/js/jqueryui/jquery-ui-1.13.2.custom/jquery-ui.min.js#L3

Reply:
I selected the plugins that were default in collective.js.jqueryui and the difference in size was small in relation to all plugins. So I changed my mind and I think it's not worth removing plugins. It would have to be documented which ones to select and would make things more difficult. Sorry for the inconvenience.